courses/CSF211/README.md

```diff
@@ -79,3 +80,4 @@ This course is a prerequisite for:
 *   [r/datastructures](https://www.reddit.com/r/datastructures/)
 *   [r/algorithms](https://www.reddit.com/r/algorithms/)
 *   [Data Structures and Algorithms, *CodeChef*](https://discuss.codechef.com/t/data-structures-and-algorithms/6599)
+*   [TheAlgorithms](https://the-algorithms.com/)
```

Comment:
This resource doesn't look like a discussion *community*. Could you replace it with another `Algorithms` community, where folks could ask questions and learn more about the domain?

Reply:
Yeah, my bad. I got it wrong, will provide the discord channel URL.